ansible/modules/hashivault/hashivault_auth_method.py

```diff
@@ -101,12 +101,8 @@ def hashivault_auth_method(module):
     elif state == 'disabled' and exists:
         changed = True
     elif exists and state == 'enabled':
-        current_state = client.sys.read_auth_method_tuning(path=mount_point)['data']
```

Comment:
I'm probably missing something here, but how does this work without the client call?